third_party/google/api/backend.proto

```diff
@@ -96,6 +96,25 @@ message BackendRule {
   string selector = 1;
 
   // The address of the API backend.
+  //
+  // The scheme is used to determine the backend protocol and security.
```

Comment:
Auto update to proto files unrelated to this? Is this just stale by now and wont be needed when you check this in?